change/@fluentui-react-skeleton-81969227-cd69-45ea-b114-38be731418ac.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore: release react-skeleton to unstable",
```

I think this is also a feat since you are moving from alpha to beta
```suggestion
  "comment": "feat: release react-skeleton to unstable",
```